### DISCUSSION TITLE
🎉 Deploy Manifest to production in no time !

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#224] [Announcements]
![Frame 587](https://github.com/user-attachments/assets/503adebb-9e2a-4bec-88dc-3630264ac5ce)

Hello everyone! 👋

It’s finally here: the first deployment documentation for Manifest! Now you can put your use cases into production super quickly and easily with this doc. We’re excited to see how you’ll use Manifest! 🎉

➡️ [Deploy Manifest](https://manifest.build/docs/deploy)

👉 Don’t forget to share what you’re building in the [Show and Tell](https://github.com/mnfst/manifest/discussions/categories/show-tell) section!

Regards

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
